Header header
int32 depth
string address
int32 sending_value
string tag
string message
bool RDY
